controllers/gslb_controller.go

```diff
@@ -85,6 +86,7 @@ func (r *GslbReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	if err != nil {
 		return result.RequeueError(fmt.Errorf("resolving spec (%s)", err))
 	}
+	log.Debug().Msgf("Resolved strategy: %s", str.ToString(gslb.Spec.Strategy))
```

Comment:
Probably should be structured as well:
```suggestion
	log.Debug().
		Str("Strategy", str.ToString(gslb.Spec.Strategy)).
		Msg("Resolved strategy")
```